xtask/src/main.rs

```diff
@@ -0,0 +1,34 @@
+use anyhow::Result;
+use clap::Parser;
+
+macro_rules! command {
+    ($enum_name:ident ( $($module:ident,)+ )) => {
+        $(mod $module;)+
+
+        #[derive(::clap::Subcommand)]
+        #[allow(non_camel_case_types)]
+        enum $enum_name {
+            $($module(crate::$module::Args),)+
+        }
+
+        impl $enum_name {
+            fn main(self) -> ::anyhow::Result<()> {
+                match self {
+                    $(Self::$module(args) => crate::$module::main(args),)+
+                }
+            }
+        }
+    }
+}
+
+command!(Command(set_version,));
```

Comment:
A short command what it does would be nice (so it displays when doing `cargo xtask --help`)